pubsub.go

```diff
@@ -43,6 +46,19 @@ type ReceiveSettings struct {
 	Settings pubsub.ReceiveSettings
 }
 
+// SubscriptionConfigs is an extension of Google PubSub's SubscriptionConfig that
+// enables further configuration for a subscription of a topic
+type SubscriptionConfigs struct {
+	EnableMessageOrdering bool
+	RetainAckedMessages   bool
+}
+
+// TopicConfigs is an extension of Google PubSub's TopicConfig that
+// enables further configuration for a topic
+type TopicConfigs struct {
```

Comment:
Same thoughts as mentioned for `SubscriptionConfigs`

Reply:
renamed TopicConfigs as suggested